### PR TITLE
camera up vector not respected when switching cameras

### PR DESCRIPTION
**Pull request by bsergean**
See https://github.com/mattdesl/three-orbit-controls/issues/9

The fix is rather simple, and should probably be submitted discussed upstream to the three.js project.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -694,7 +700,7 @@ module.exports = function(THREE) {
         this.domElement.addEventListener( 'contextmenu', function ( event ) { event.preventDefault(); }, false );
         this.domElement.addEventListener( 'mousedown', onMouseDown, false );
         this.domElement.addEventListener( 'mousewheel', onMouseWheel, false );
-        this.domElement.addEventListener( 'DOMMouseScroll', onMouseWheel, false ); // firefox
+        this.domElement.addEventListener( 'MozMousePixelScroll', onMouseWheel, false ); // firefox
```
Comment (bsergean) [inline index.js:704]: Argh this is from my previous patch request ... let me try to do the git magic (rebase ?) to make this go away.

**Comment by bsergean** [comment]
@mattdesl / do you mind if I push this change as a stop gap while the sync to r73 (aka https://github.com/mattdesl/three-orbit-controls/issues/11) is being worked out ?

**Comment by bsergean** [comment]
(adding that r72 and r73 are a bit disruptive version of three.js so my internal project cannot jump to them right away).

**Comment by mattdesl** [comment]
Thanks. I merged this, and cleaned it up a little to re-use the same quaternions instead of creating new ones per frame. This should minimize GC thrashing. Let me know if my changes work for you. :+1:

**Pull request merged**